cmd/kpod/import.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	importFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "change, c",
+			Usage: "Apply the following possible instructions to the created image (default []): CMD | ENTRYPOINT | ENV | EXPOSE | LABEL | STOPSIGNAL | USER | VOLUME | WORKDIR",
+		},
+		cli.StringFlag{
+			Name:  "message, m",
+			Usage: "Set commit message for imported image",
+		},
+	}
+	importDescription = `Create a container image from the contents of the specified tarball (.tar, .tar.gz, .tgz, .bzip, .tar.xz, .txz).
+	 Note remote tar balls can be specified, via web address.
+	 Optionally tag the image. You can specify the Dockerfile instructions using the --change option.
+	`
+	importCommand = cli.Command{
+		Name:        "import",
+		Usage:       "Import a tarball to create a filesystem image",
+		Description: importDescription,
+		Flags:       importFlags,
+		Action:      importCmd,
+		ArgsUsage:   "TARBALL [REFERENCE]",
+	}
+)
+
+func importCmd(c *cli.Context) error {
+	if err := validateFlags(c, importFlags); err != nil {
+		return err
+	}
+
+	runtime, err := getRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "Could not get runtime")
```

Comment:
Errors should start with a lowercase letter.

@rhatdan Golint should have caught this - are we running it in the tests yet?

Reply:
fixed.